recipes/fasd/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "fasd" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clvv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 88efdfbbed8df408699a14fa6c567450bf86480f5ff3dde42d0b3e1dee731f65
+
+build:
+  number: 0
+  script: "make install"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - make
+  host:
```

Comment:
This should be a run dep, not a host one, right?

Reply:
> This should be a run dep, not a host one, right?

Thanks for review. I don't think we should put it as runtime deps. My understanding it'll pack bash to target to install. 

I double checked fasd script and it just uses shell in host: https://github.com/clvv/fasd/blob/master/fasd#L1
For some existing packages, e.g. luarocks. It's using shell as well but no such deps.

Maybe we should remove either bash or zsh deps as luarocks did?

Reply:
This needs a sh-lang to actually use right?

Reply:
yes, but it's either `/usr/bin/env sh` or `/usr/bin/sh`. not conda's bash.

Reply:
`/usr/bin/env sh` can point to the bash in the conda environment.

Reply:
yes it can. but it depends on target's sh config `/etc/shells`. Usually not conda's bash. `run` deps install conda's `bash` on target. But no use unless they `chsh`. So bash shouldn't be deps for conda install. The fasd script is not intended to use particular conda's shell to do its job but host's sh.